test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -427,3 +427,73 @@ Feature: Bind an application to a service using annotations
                 ready: true
             """
         Then Secret "binding-request-backend-ann" is empty
+
+
+    @disabled
+    # Verify https://github.com/redhat-developer/service-binding-operator/issues/838
+    Scenario: Verify data of secret created by service binding
```

Comment:
This scenario is referring a key from a config map and for that we have dedicated feature file: `bindAppToServiceUsingConfigMap.feature`. Even more, we have scenario there that should actually cover it:

https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToServiceUsingConfigMap.feature#L10

but it looks that it uses a wrong annotation. Hence that scenario should be updated, instead of adding a new one. Similar scenario exists in 

https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/bindAppToServiceUsingSecret.feature#L10

That should be also updated, because the spec defines that `sourceKey` should be used:

https://github.com/k8s-service-bindings/spec#annotation-examples

```
“service.binding/certificate”:
  "path={.status.data.dbConfiguration},objectType=Secret,sourceKey=certificate"
```

Reply:
> This scenario is referring a key from a config map and for that we have dedicated feature file: `bindAppToServiceUsingConfigMap.feature`. Even more, we have scenario there that should actually cover it:
> 

Actually, this behaviour was due to annotation which is same if I even use secrets so I put this test in the annotation file.

> but it looks that it uses a wrong annotation. Hence that scenario should be updated, instead of adding a new one. Similar scenario exists in

I will update other scenarios as well but this test will serve as a regression test for  #838. So I think this test should be in test suites.

Reply:
> I will update other scenarios as well but this test will serve as a regression test for #838. So I think this test should be in the test suite.

When you update these scenarios, there is no need to add additional scenario, because they are covering the same issue. or are they different and how?

Reply:
In this scenario, only one step is more which is testing regression, https://github.com/redhat-developer/service-binding-operator/pull/842/files#diff-4af525dbccc1a8950ce632ed0a0372567bd9159532d9a3a04349a1ee5ffe8170R495.

Reply:
But I'll update the existing scenario with this step.

Reply:
You do not need to update the existing scenario at all. The step

```
Secret "cmsa-1" does not have key "BACKEND"
```

is not relevant, since the only thing what counts is what the application sees. The content of the secret is an implementatino detail, and not something what user should be aware of.